src/py_class/py_class_impl2.rs

```diff
@@ -487,21 +487,21 @@ macro_rules! py_class_impl {
             $type_slots
             /* as_number */ [
                 $( $nb_slot_name : $nb_slot_value, )*
-                nb_nonzero: $crate::py_class_unary_slot!($class::__bool__, $crate::_detail::libc::c_int, $crate::py_class::slots::BoolConverter),
+                nb_bool: $crate::py_class_unary_slot!($class::__bool__, $crate::_detail::libc::c_int, $crate::py_class::slots::BoolConverter),
```

Comment:
This is the source of at least one error.  The `py_class_impl2.rs` script needs to be run with env `PY=2` in order to generate correct output.  (You should get this if you use `make` to generate them.  Did you run the script manually?)

Reply:
I did. It wasn't obvious to me how to regenerate the files so I ran the script manually. I'll regenerate with `make` and then adds some docs.

Reply:
Okay, pushed a fix with those files regenerated via the Makefile.

Reply:
Thanks!